src/components/SearchForm/SearchForm.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ChangeEvent, useState } from 'react';
+import './SearchForm.scss';
+
+interface Props {
```

Comment:
Sometimes it is necessary to export an interface, in this case it is better to have unique names all over the app for consistency.

```suggestion
interface SearchFormProps {
```